src/game_interpreter.cpp

```diff
@@ -957,15 +957,11 @@ bool Game_Interpreter::CommandControlVariables(RPG::EventCommand const& com) { /
 			break;
 		case 1:
 			// Var A ops B
-			if (com.parameters[0] != 1) {
-				value = Main_Data::game_variables->Get(com.parameters[5]);
-			}
+			value = Main_Data::game_variables->Get(com.parameters[5]);
```

Comment:
Always interesting to see things getting back to their former state.

Reply:
This was a silly optimization gone wrong.

Reply:
This is why we need a test suite. Would have catched this. Maybe, is an edge case. 🤔

Reply:
Even if the test suite didn't catch it, once the bug is reported we make a new test and it never happens again no matter how many ways we try to refactor or optimize the underlying code.